File: C/Iniciante/1051impostoderenda.c
#include <stdio.h>
int main() {
    double salario;
    scanf("%lf",&salario);
    if(salario>=0 && salario<=2000){
        printf("Isento\n");
    }
    if(salario>2000 && salario<=3000){
        salario-=2000;
        printf("R$ %.2lf\n",(salario*0.08));
    }
    if(salario>3000 && salario<=4500){
        salario-=3000;
        printf("R$ %.2lf\n",((1000*0.08)+(salario*0.18)));
    }
    if(salario>4500){
        salario-=4500;
        printf("R$ %.2lf\n",((1000*0.08)+(1500*0.18)+(salario*0.28)));
    }
    return 0;
}

File: Python/iniciante/1051impostoderenda.py
salario = float(input())
if 0 <= salario <= 2000:
    print("Isento")
elif 2000 < salario <= 3000:
    salario -= 2000
    print("R$ %.2lf" % (salario * 0.08))
elif 3000 < salario <= 4500:
    salario -= 3000
    print("R$ %.2lf" % ((1000 * 0.08) + (salario * 0.18)))
elif 4500 < salario:
    salario -= 4500
    print("R$ %.2lf" % ((1000 * 0.08) + (1500 * 0.18) + (salario * 0.28)))
